Amazon.QLDB.Driver/driver/AsyncQldbDriverBuilder.cs

```diff
@@ -30,7 +28,7 @@ internal AsyncQldbDriverBuilder()
 
         private protected override AsyncQldbDriverBuilder BuilderInstance => this;
 
-        private protected override string UserAgentStringPrefix => "Async QLDB Driver for .NET v";
+        private protected override string UserAgentStringPrefix => "Async " + base.UserAgentStringPrefix;
```

Comment:
The "Async" part should be a an "extension", not change the name of the user agent.